src/Equations/damaged-elastic/Kernels/Local.cpp

```diff
@@ -515,3 +501,565 @@ unsigned seissol::kernels::Local::bytesIntegral() {
+                              std::sqrt(EspIIp)) +
+          qIPlus[o * seissol::dr::misc::numQuantities * seissol::dr::misc::numPaddedPoints +
+                 BRE * seissol::dr::misc::numPaddedPoints + i] *
+              (2.0 * aB2 + 3.0 * xip * aB3 +
+               aB1 *
+                   (qIPlus[o * seissol::dr::misc::numQuantities *
+                               seissol::dr::misc::numPaddedPoints +
+                           XX * seissol::dr::misc::numPaddedPoints + i] +
+                    epsInitxx) /
+                   std::sqrt(EspIIp));
+      real mup =
+          (1 - qIPlus[o * seissol::dr::misc::numQuantities * seissol::dr::misc::numPaddedPoints +
+                      BRE * seissol::dr::misc::numPaddedPoints + i]) *
+              (mu0P - alphap * materialData[l_cell].local.xi0 * materialData[l_cell].local.gammaR -
+               0.5 * alphap * materialData[l_cell].local.gammaR * xip) +
+          qIPlus[o * seissol::dr::misc::numQuantities * seissol::dr::misc::numPaddedPoints +
+                 BRE * seissol::dr::misc::numPaddedPoints + i] *
+              (aB0 + 0.5 * xip * aB1 - 0.5 * xip * xip * xip * aB3);
+
+      real lambm =
+          (1 - qIMinus[o * seissol::dr::misc::numQuantities * seissol::dr::misc::numPaddedPoints +
+                       BRE * seissol::dr::misc::numPaddedPoints + i]) *
+              (lambda0M - alpham * materialData[l_cell].neighbor[side].gammaR *
+                              (qIMinus[o * seissol::dr::misc::numQuantities *
+                                           seissol::dr::misc::numPaddedPoints +
+                                       XX * seissol::dr::misc::numPaddedPoints + i] +
+                               epsInitxx) /
+                              std::sqrt(EspIIm)) +
+          qIMinus[o * seissol::dr::misc::numQuantities * seissol::dr::misc::numPaddedPoints +
+                  BRE * seissol::dr::misc::numPaddedPoints + i] *
+              (2.0 * aB2 + 3.0 * xim * aB3 +
+               aB1 *
+                   (qIMinus[o * seissol::dr::misc::numQuantities *
+                                seissol::dr::misc::numPaddedPoints +
+                            XX * seissol::dr::misc::numPaddedPoints + i] +
+                    epsInitxx) /
+                   std::sqrt(EspIIm));
+
+      real mum =
+          (1 - qIMinus[o * seissol::dr::misc::numQuantities * seissol::dr::misc::numPaddedPoints +
+                       BRE * seissol::dr::misc::numPaddedPoints + i]) *
+              (mu0M -
+               alpham * materialData[l_cell].neighbor[side].xi0 *
+                   materialData[l_cell].neighbor[side].gammaR -
+               0.5 * alpham * materialData[l_cell].neighbor[side].gammaR * xim) +
+          qIMinus[o * seissol::dr::misc::numQuantities * seissol::dr::misc::numPaddedPoints +
+                  BRE * seissol::dr::misc::numPaddedPoints + i] *
+              (aB0 + 0.5 * xim * aB1 - 0.5 * xim * xim * xim * aB3);
```

Comment:
Could also be extracted into two individual functions.

Reply:
I extracted it into one function and called it twice.